src/scss/_main.scss

```diff
@@ -464,6 +464,11 @@
 			width: oGridGetMaxWidthForLayout($o-grid-fixed-layout, $grid-mode: 'fixed');
 		}
 	}
+
+	&--bleed {
```

Comment:
But how do I use this in Sass? Shouldn't the bleed should be a parameter, and this class be generated separately as part of the "generate" mixin?